ginga/misc/plugins/MultiDim.py

```diff
@@ -254,6 +260,27 @@ def build_naxis(self, dims):
         # Add vbox of naxis controls to gui
         self.naxisfr.set_widget(w)
 
+        # for storing play_idx for each dim of image. used for going back to
+        # the idx where you left off.
+        self.play_indices = [0 for i in range(len(dims) - 2)]
+
+        def play_axis_change_func_creator(n):
```

Comment:
Doesn't matter much unless you are capturing lexical environment for use in the function.